pallets/rc-migrator/src/lib.rs

```diff
@@ -0,0 +1,2581 @@
+		/// - create_call - function to create the call from the items
+		///
+		/// Will modify storage in the error path.
+		/// This is done to avoid exceeding the XCM message size limit.
+		pub fn send_chunked_xcm_and_track<E: Encode>(
+			items: impl Into<XcmBatch<E>>,
+			create_call: impl Fn(Vec<E>) -> types::AhMigratorCall<T>,
+		) -> Result<u32, Error<T>> {
+			let mut items = items.into();
+			log::info!(target: LOG_TARGET, "Batching {} items to send via XCM", items.len());
+			defensive_assert!(items.len() > 0, "Sending XCM with empty items");
+			let mut batch_count = 0;
+
+			while let Some(batch) = items.pop_front() {
+				let batch_len = batch.len() as u32;
+				log::info!(target: LOG_TARGET, "Sending XCM batch of {} items", batch_len);
+
+				let asset_hub_location = Location::new(0, Parachain(1000));
+
+				let receive_notification_call =
+					Call::<T>::receive_query_response { query_id: 0, response: Default::default() };
+
+				let query_id = pallet_xcm::Pallet::<T>::new_notify_query(
+					asset_hub_location.clone(),
+					<T as Config>::RuntimeCall::from(receive_notification_call),
+					frame_system::Pallet::<T>::block_number() + T::XcmResponseTimeout::get(),
+					Location::here(),
+				);
+
+				let call = types::AssetHubPalletConfig::<T>::AhmController(create_call(batch));
+				let message = vec![
+					Instruction::UnpaidExecution {
+						weight_limit: WeightLimit::Unlimited,
+						check_origin: None,
+					},
+					Instruction::Transact {
+						origin_kind: OriginKind::Superuser,
+						fallback_max_weight: None,
+						call: call.encode().into(),
+					},
+				];
+
+				let message_hash = T::Hashing::hash_of(&message);
+
+				let message_with_report = {
+					let mut m = message.clone();
+					m.push(SetAppendix(Xcm(vec![ReportTransactStatus(QueryResponseInfo {
+						destination: Location::parent(),
+						query_id,
+						max_weight: T::RcWeightInfo::receive_query_response(),
+					})])));
+					m
+				};
+
+				if let Err(err) =
+					send_xcm::<T::SendXcm>(asset_hub_location, Xcm(message_with_report))
+				{
+					log::error!(target: LOG_TARGET, "Error while sending XCM message: {:?}", err);
+					return Err(Error::XcmError);
+				} else {
+					PendingXcmMessages::<T>::insert(message_hash, Xcm(message));
+					PendingXcmQueries::<T>::insert(query_id, message_hash);
+					batch_count += 1;
+				}
+			}
+
+			if batch_count > MAX_XCM_MSG_PER_BLOCK {
+				log::warn!(
+					target: LOG_TARGET,
+					"Maximum number of XCM messages ({}) to migrate per block exceeded, current msg count: {}",
+					MAX_XCM_MSG_PER_BLOCK,
+					batch_count
+				);
+			}
+
+			log::info!(target: LOG_TARGET, "Sent {} XCM batch/es", batch_count);
+			Ok(batch_count)
+		}
+
+		/// Send a single XCM message.
+		///
+		/// ### Parameters:
+		/// - call - the call to send
+		pub fn send_xcm(call: types::AhMigratorCall<T>) -> Result<(), Error<T>> {
+			log::info!(target: LOG_TARGET, "Sending XCM message");
```

Comment:
Pretty pointless log to take precious space in `info` target :D 

@ggwpez @muharem I think now is actually perfect time for you two to revisit all logs, and see which ones are useful for you, and move them to info. Rest can be debug/trace based on frequency. 

Note: AHM-Dryrun repo's CI gives you current logs of the migration.

Reply:
I know that Muharem actually did this in the past, but in this case we can delete it, yes.